posts/2019/11/Files-Improvements.md

```diff
@@ -0,0 +1,77 @@
+
+## Simplified interface
+
+In order to simplify the file explorer interface, by default now the columns corresponding to size and kind are hidden. This feature can be activated or deactivated by either accessing the Pane Menu or by right clicking the header directly. In both, cases you can select which columns to display in the file explorer.
+
+![Pane Menu](/images/spyder-files-pane-menu.png)
+<!-- TEASER_END -->
+
+![Header Menu](/images/spyder-files-header-menu.png)
+
+## Custom file associations
+
+First, we added the possibility to associate different external applications to open specific file extensions. By going to the “Files” option in the preferences window, it is possible to set the file associations by, selecting the File Types and the Associated applications that should be used to open this specific file extensions. 
+
+![File associations](/images/spyder-files-file-associations.png)
+
+Once this configuration is set you can go to the files pane in Spyder and whenever you open a file with the selected extension, it will open automatically with the selected associated application. Additionally, when you right click a file with one of this extensions, you will find an “open with” option that will show the application associated for this extension.
+
+![Open With](/images/spyder-files-open-with.png)
+
+## Single click open on file explorer
+
+Another improvement made was the option to open files and directories with a single click which was not possible before because files and folders could only be opened by double clicking them. In order to enable this option, go to the files tab in the preferences window, check the box “Single click to open files” and apply the changes. With this set, now only a single click is needed to open a file externally or in Spyder, from the files tab. 
+
+![Single click](/images/spyder-files-single-click.png)
+
+Bear in mind that changing this configuration option will also affect the behaviour of the Project Explorer pane.
+
+## Open files externally
+
+Additionally, we added a context menu action called Open externally to open files with the Operating System default program associated with the file type. To use this action, go to the file you want to open in the files tab and right-click it to see the context menu. Then select “Open Externally” and the file will be opened outside Spyder, with the default program associated with its file type, depending on the Operating System.
+
+![Open Externally](/images/spyder-files-open-externally.png)
+
+Moreover, now it is possible to select several files to perform a specific action thanks to the multi-select functionality added. For this, press CTRL or SHIFT and Click on the files that are going to be selected without releasing the CTRL/SHIFT. It is also possible to select the files without doing it one by one, but selecting a file with the CTRL/SHIFT clicked and selecting a not contiguous file to this one which will select all the files between them.
```

Comment:
```suggestion
Moreover, it is now possible to select more than one file on which to perform a context menu action, thanks to the Files pane's new multi-select functionality. For this, press `Ctrl` (`Cmd` on macOS) or `Shift` while clicking on the files to select, just as you would in your operating system's file manager (Windows Explorer, macOS Finder, etc). As you'd expect, `Ctrl` (`Cmd`) selects individual files one by one, while `Shift` selects all the files between the one you click and the one previously selected.
```

* Make keyboard shortcuts verbatim, title case and cross platform
* Fix capitalization
* Clarify and simplify explanations and remove redundancy and awkwardness
* More clearly and accurately explain behavior
* Reinforce standard user expectations from using their regular file manager
* Fix grammar
* Use proper UI names